Locales/frFR.lua

```diff
@@ -32,7 +32,8 @@ L["Module Category AzerothianArchives"] = "Archives d'Azeroth";     --Added in p
 
 --AutoJoinEvents
 L["ModuleName AutoJoinEvents"] = "Rejoindre automatiquement les événements";
-L["ModuleDescription AutoJoinEvents"] = "Sélectionne automatiquement (Commencer la Faille Temporelle) lorsque vous interagissez avec Soridormi pendant l'événement.";
+L["ModuleDescription AutoJoinEvents"] = "Auto join these events when you interact with the NPC: \n\n- Time Rift\n\n- Theater Troupe";
+L["ModuleDescription AutoJoinEvents"] = "Rejoint automatiquement l'évènement lorsque vous intéragissez avec le PNJ : : \n\n- Faille temporelle\n\n- Groupe de théâtre";
```

Comment:
@KatyPurry, it seems like the English string was inserted by mistake?

Reply:
yes It was another mistake, sorry